src/next-transpile-modules.js

```diff
@@ -87,32 +87,35 @@ const withTmInitializer = (transpileModules = []) => {
         // TODO ask Next.js maintainer to expose the css-loader via defaultLoaders
         const nextCssLoaders = config.module.rules.find((rule) => typeof rule.oneOf === 'object');
 
-        // .module.css
+        // .module.css and .module.css
         if (nextCssLoaders) {
           const nextCssLoader = nextCssLoaders.oneOf.find(
             (rule) => rule.sideEffects === false && regexEqual(rule.test, /\.module\.css$/)
           );
-
           if (nextCssLoader) {
             nextCssLoader.issuer.include = nextCssLoader.issuer.include.concat(includes);
             nextCssLoader.issuer.exclude = excludes;
           }
 
+          const nextSassLoader = nextCssLoaders.oneOf.find(
+            (rule) => rule.sideEffects === false && regexEqual(rule.test, /\.module\.(scss|sass)$/)
+          );
+
+          if (nextSassLoader) {
+            nextSassLoader.issuer.include = nextSassLoader.issuer.include.concat(includes);
+            nextSassLoader.issuer.exclude = excludes;
+          }
+
           // Hack our way to disable errors on node_modules CSS modules
           const nextErrorCssLoader = nextCssLoaders.oneOf.find(
-            (rule) => rule.use && rule.use.loader === 'error-loader' && regexEqual(rule.test, /\.module\.css$/)
+            (rule) => rule.use && rule.use.loader === 'error-loader' && (Array.isArray(rule.test) ? regexEqual(rule.test[0], /\.module\.css$/) : regexEqual(rule.test, /\.module\.css$/))
```

Comment:
the `[0]` should be avoided imo, if we can find a more dynamic way to do it, that'd be great

Reply:
It is either one regexp, that catches .module.css, or array of regexps, one of which captures it.

Either way function will be long if it would be written in such way. Probably that part of code should be written with ifs to be more readable.

Reply:
Other way is to do 
```
rule.some((regex) => regexEqual(regex,  /\.module\.css$/))
```